indra/llappearance/llavatarappearancedefines.cpp

```diff
@@ -252,6 +252,7 @@ LLAvatarAppearanceDictionary::BakedEntry::BakedEntry(ETextureIndex tex_index,
         LLWearableType::EType t = (LLWearableType::EType)va_arg(argp,int);
         mWearables.push_back(t);
     }
+    va_end(argp);
```

Comment:
Ha, I had a momentary thought of wrapping `va_list` in a class with a destructor -- but that effort would be better spent converting the consuming function to variadic (if it could be moved to `.h`) or to accept (e.g.) a `std::vector` (if not).

Meanwhile, I'm okay with this improvement.